src/converse-rosterview.js

```diff
@@ -480,7 +506,11 @@ converse.plugins.add('converse-rosterview', {
                         'desc_chat': __('Click to chat with %1$s (JID: %2$s)', display_name, item.get('jid')),
                         'desc_remove': __('Click to remove %1$s as a contact', display_name),
                         'allow_contact_removal': _converse.allow_contact_removal,
-                        'num_unread': item.get('num_unread') || 0
+                        'num_unread': item.get('num_unread') || 0,
+                        classes: '',
+                        width: '24',
+                        height: '24',
```

Comment:
Looks like the `width` and `height` values you're passing in here never get used.

Reply:
I'll remove them then.